bwmenu

```diff
@@ -71,7 +77,7 @@ rofi_menu() {
     -kb-custom-9 alt+L
   )
 
-  msg="<b>Alt+r</b>: sync      | <b>Alt+u</b>: urls      | <b>Alt+n</b>: names     | <b>Alt+c</b>: folders     | <b>Alt+t</b>: totp     | <b>Alt+L</b>: lock"
+  msg="<b>Alt+r</b>: sync      | <b>Alt+u</b>: urls      | <b>Alt+n</b>: names     | <b>Alt+c</b>: folders     | <b>Alt+t</b>: totp      | <b>Alt+L</b>: lock"
```

Comment:
I think this is not a useful change, correct? :stuck_out_tongue:

Reply:
![rsz_2019-05-18-185310_1920x1080_scrot](https://user-images.githubusercontent.com/8048511/57972909-d104ed00-79a0-11e9-9042-03808fc18228.png)
![rsz_2019-05-18-185403_1920x1080_scrot](https://user-images.githubusercontent.com/8048511/57972910-da8e5500-79a0-11e9-920b-04817a66fcec.png)
Depends on your OCD levels :)

Reply:
Actually depends on the theme you're using :p maybe we should find a better way of 'stacking' the help lines